src/queryClienConfig.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
```

Comment:
Why are you importing `QueryClientProvider` here and exporting it and then using it in the `main.tsx`

Why not import it there directly in `main.tsx`

Plus there is a typo in the file name. (Forgot the t)

Reply:
- Thank you for your review ❤

- I did that on purpose To preserve the 

> main.tsx

As clean as possible in case we add alot of libraries, That is the preferred way as suggested by ChatGPT 

- For the typo I will fix it ASAP.

Reply:
That doesn't make any sense to me,
Because you are doing the import either way
The only difference is that it will be on a separate line, which will not make the code smelly or clean